src/translator/vocabs.h

```diff
@@ -0,0 +1,68 @@
+#pragma once
+
+namespace marian {
+namespace bergamot {
+
+class VocabsGenerator {
```

Comment:
This class can be `Vocabs`.

Reply:
Generator is more appropriate name as it generates the vocab class.

Reply:
What I'm suggesting is a change to a multi-Vocabs container with knowledge of which ones are source vocabularies and which ones are target vocabularies. Remember, marian has `Vocab` and `vector<Ptr<Vocab const>>`. I want to replace `vector<Ptr<Vocab const>>` with this class.

Reply:
The reason marian has those two is because you can have multiple source and one target vocabularies as discussed elsewhere.